.github/workflows/ci_mac_gtk.yml

```diff
@@ -0,0 +1,190 @@
+            echo '-----------------------------------------------------------'
+            cat config.log
+            echo '-----------------------------------------------------------'
+            exit $rc
+          fi
+  
+      - name: Building
+        run: |
+          make -k $wxBUILD_ARGS "CXXFLAGS=$wxMAKEFILE_ERROR_CXXFLAGS"
+  
+      - name: Building tests
+        run: |
+          make -C tests $wxBUILD_ARGS failtest
+          make -k -C tests $wxBUILD_ARGS "CXXFLAGS=$wxMAKEFILE_CXXFLAGS" "LDFLAGS=$wxMAKEFILE_LDFLAGS"
+  
+      - name: Setup Go
+        if: matrix.skip_testing != true
+        uses: actions/setup-go@v5
+        with:
+          go-version: '>=1.21.0'
+          cache: false
+  
+      - name: Testing
+        if: matrix.skip_testing != true
+        working-directory: tests
+        run: |
+          . ../build/tools/httpbin.sh
+          httpbin_launch
+          export ASAN_OPTIONS=fast_unwind_on_malloc=0
+          # Explicitly use bash because /bin/sh doesn't have pipefail option
+          /bin/bash -o pipefail -c './test 2>&1 | tee test.out' || rc=$?
+          if [ -n "$rc" ]; then
+            httpbin_show_log
+            exit $rc
+          fi
+  
+      - name: Testing GUI
+        if: matrix.skip_testing != true
+        working-directory: tests
+        run: |
+          # We currently need to disable container overflow detection as we get
+          # what seems to be a false positive in BitmapComboBoxTestCase triggered
+          # by creating a new string from std::allocator<wxString>::construct()
+          # used by std::vector<>::insert().
+          export ASAN_OPTIONS='fast_unwind_on_malloc=0 detect_container_overflow=0'
+          # Exclude tests that are currently known to fail
+          wx_tests_selection='~[.] ~[valnum] ~wxPersistTLW ~wxPersistDVC ~wxTopLevel::Show ~wxFont::Weight ~wxFont::NativeFontInfo ~WebView ~RadioButton::Single ~RadioButton::Focus ~wxHtmlPrintout::Pagination ~wxTextCtrl::GetBestSize ~TextCtrlTestCase ~wxExecute::RedirectUTF8 ~Ellipsization::NormalCase ~SpinCtrl::* ~SpinCtrlDouble::* ~NotebookTestCase ~SettingsTestCase ~Window::Show ~ModalDialogsTestCase ~Button::Click ~Button::Disabled ~wxDVC::GetItemRect ~wxDVC::AppendTextColumn ~Grid::KeyboardSelection ~Grid::CellClick ~Grid::ReorderedColumnsCellClick ~Grid::CellSelect ~wxStyledTextCtrl::AutoComp ~EvtLoopTestCase ~EventPropagationTestCase ~wxTreeCtrl::ItemClick ~wxTreeCtrl::LabelEdit ~wxTreeCtrl::KeyDown ~wxTreeCtrl::CollapseExpandEvents ~wxTreeCtrl::SelectionChange ~wxTreeCtrl::SelectItemMultiInteractive ~wxTreeCtrl::Menu ~wxTreeCtrl::KeyNavigation'
+          ./test_gui.app/Contents/MacOS/test_gui $wx_tests_selection
+  
+      - name: Building samples
+        if: matrix.skip_testing != true && matrix.skip_samples != true
+        run: |
+          make -k $wxBUILD_ARGS "CXXFLAGS=$wxMAKEFILE_CXXFLAGS" "LDFLAGS=$wxMAKEFILE_LDFLAGS" samples
+  
+      - name: Installing
+        if: matrix.skip_testing != true
```

Comment:
I think we should always test installation and compiling `minimal` using the installed version, this doesn't take much time and really should work.

I.e. I'd remove this `if:` and the one below.

Reply:
Thanks, did that